src/libs/azure-utils.js

```diff
@@ -35,6 +35,8 @@ export const azureMachineTypes = {
 export const getMachineTypeLabel = (key) =>
   _.has(key, azureMachineTypes) ? `${key}, ${azureMachineTypes[key].cpu} CPU(s), ${azureMachineTypes[key].ramInGb} GBs` : 'Unknown machine type';
 
+export const machineTypeHasGpu = (machineType) => !!azureMachineTypes[machineType]?.hasGpu;
```

Comment:
I know it was like this already, but consider converting this file to TS and typing machineType while you're in here if there's not too much complaining from the existing code

Reply:
To avoid holding this PR up, I'll do that in a follow up.